src/mono/wasm/runtime-test.js

```diff
@@ -4,21 +4,19 @@
 //
 
 //glue code to deal with the differences between chrome, ch, d8, jsc and sm.
-var is_browser = typeof window != "undefined";
+const is_browser = typeof window != "undefined";
+const is_node = !is_browser && typeof process != 'undefined';
 
 // if the engine doesn't provide a console
-if (typeof (console) === "undefined") {
-	var console = {
+if (typeof (console) === "undefined"){
+	console = {
 		log: globalThis.print,
```

Comment:
there's code in runtime-test.js and in our support code that uses console.error, so we need to define console.error as well

Reply:
We define `console.error` later (line 36) but in terms of console.log. So this snippet is defining `console.log` is it doesn't exist so when we get to line 36 we can guarantee having it.
```javascript
const methods = ["debug", "trace", "warn", "info", "error"];
for (let m of methods) {
	if (typeof(console[m]) !== "function") {
		console[m] = proxyMethod(`console.${m}: `, console.log, false);
	}
}
```

Reply:
I see, looks good then